### PR TITLE
Decoupling directory endpoints as per Magedoc3974

### PR DESCRIPTION
**Pull request by erikmarr**
## Purpose of this pull request

This pull request (PR) decouples the countries, country, and currency endpoints into separate topics.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/graphql/reference/directory.html

### CODE REVIEW

File: guides/v2.3/graphql/queries/directory-countries.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+        ]
+      }
+    ]
+  }
+}
+```
+
+## Input attributes
+
+The `countries` object provides the following attributes:
```
Comment (keharper) [inline guides/v2.3/graphql/queries/directory-countries.md]:
```suggestion
The query returns an array of `Country` objects.
```

File: guides/v2.3/graphql/queries/directory-countries.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+Attribute | Data type | Description
+--- | --- | ---
+`Country` | [[Country]]({{page.baseurl}}/graphql/queries/directory-country.html#inputAttributes)| Provides information for all countries specified in the store
```
Comment (keharper) [inline guides/v2.3/graphql/queries/directory-countries.md]: Delete this table. Replace with a reference to an include file.

File: guides/v2.3/graphql/queries/directory-country.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+          "name": "Western Australia"
+        }
+      ]
+    }
+  }
+}
+```
+
+## Input attributes {#inputAttributes}
+
```
Comment (keharper) [inline guides/v2.3/graphql/queries/directory-country.md]: The Input attributes section is limited to `id`

The Output attributes section should say "The query returns a single `Country` object. Then add a reference to the include file.

File: guides/v2.3/graphql/queries/directory-country.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+Attribute | Data type | Description
+--- | --- | ---
+`available_regions` | [[Region]](#region) | An array of regions within a particular country
+`full_name_english` | String | The name of the country in English
+`full_name_locale` | String | The locale name of the country
+`id` | String | A unique ID for the country
+`three_letter_abbreviation` | String | The three-letter abbreviation of the country, such as `USA`
+`two_letter_abbreviation` | String | The two-letter abbreviation of the country, such as `US`
+
+## Region attributes {#region}
```
Comment (keharper) [inline guides/v2.3/graphql/queries/directory-country.md]:
```suggestion
### Region attributes {#region}
```

File: guides/v2.3/graphql/queries/directory-country.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+        }
+      ]
+    }
+  }
+}
+```
+
+## Input attributes {#inputAttributes}
+
+The `country` object provides the following attributes:
```
Comment (keharper) [inline guides/v2.3/graphql/queries/directory-country.md]:
```suggestion
The `Country` object provides the following attributes:
```

File: guides/v2.3/graphql/queries/directory-country.md (hunk 3 above)
Comment (erikmarr) [reply guides/v2.3/graphql/queries/directory-country.md]: I put the description plus headers in the include file.

Comment (erikmarr) [reply guides/v2.3/graphql/queries/directory-country.md]: Cleaned this up after talking to Kevin.

**Comment by erikmarr** [comment]
running tests

**Pull request merged**